src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -789,6 +796,26 @@ def pickRSE(self, rseExpression='rse_type=TAPE\cms_type=test', rseAttribute='ddm
         choice = weightedChoice(rsesWithWeights)
         return choice
 
+    def requiresApproval(self, rse):
+        """
+        _requiresApproval_
+
+        Returns wether or not the specified RSE requires rule approval.
+        :param res: Rucio RSE name
+        :returns: True if the RSE requires approval to write (rule property)
+        """
+        
+        try:
+            attrs = self.cli.list_rse_attributes(rse)
+        except RSENotFound as exc:
+            msg = "RSE not found in Rucio: {}. Error: {}".format(rse, str(exc))
+            raise WMRucioRSENotFound(msg)
+
+        requiresApproval = attrs.get('requires_approval', False)
```

Comment:
I'm inclined to think that having a default value to `True` would be safer, no?

Reply:
The thing is, sites that don't require approval don't have a `requires_approval` attribute. At least I have never seen 'requires_approval=False'. It is either 'requires_approval=True' or not 'requires_approval' at all.

Reply:
Gotcha! Then default to `False` makes sense.